test/abnormal-test/test-FuseCode.js

```diff
@@ -0,0 +1,703 @@
+describe('Abnormal Test', function() {
+  const assert = chai.assert;
+  let nn, operandIndex;
+
+  beforeEach(function(){
+    nn = navigator.ml.getNeuralNetworkContext();
+    operandIndex = 0;
+  });
+
+  describe('#addOperation API', function() {
+    it('Fuse code as "4" for operation code "ADD"', function() {
```

Comment:
The purpose of test case is not obvious, please use 'raise error when the value of input2 is invalid(out of 0-3) for  "ADD" operation'